packages/runtime/container-runtime/src/gcSweepReadyUsageDetection.ts

```diff
@@ -0,0 +1,132 @@
+/*!
```

Comment:
Not part of this PR - We should move GC files into its own folder. Then we can break more functionality out of the main GC file. There are parts that should be in separate file but I did not do it because I didn't want too many files under container-runtime/src folder.

Reply:
Yeah I took a middle-ground of staying in the same folder but putting gc prefix - they'll all be adjacent in the directory.  Hesitating on doing the more disruptive directory while we're at a critical time in rolling this out - if we want to be patching between releases it'll make it more painful (even moving code makes it a little painful, but felt worth it).

There are some things to consider when adding sub-dirs, I made a post about it in the Dev channel while you were OOF. Don't remember what they are.  Maybe we open a backlog item for it, it will take some effort.